android/app/build.gradle

```diff
@@ -50,6 +50,21 @@ android {
         versionName flutterVersionName
     }
 
+    flavorDimensions "stage"
+
+    productFlavors {
+        production {
+            dimension "stage"
+            applicationId "co.nimblehq.survey"
+            resValue "string", "app_name", "Survey"
+        }
+        staging {
+            dimension "stage"
+            applicationId "co.nimblehq.survey.staging"
+            resValue "string", "app_name", "Survey Staging"
```

Comment:
@chornerman we should use strings.xml in each flavor to define app names for each env instead

Reply:
@luongvo Updated 👍🏼